composer.json

```diff
@@ -10,37 +10,41 @@
     ],
     "require": {
         "php": "^7",
-        "symfony/symfony": "^3.4",
-        "twig/twig": "^2",
+        "symfony/console": "^4.4",
+        "twig/twig": "^2 || ^3",
         "zicht/util": "~1.1"
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7",
-        "sonata-project/admin-bundle": "*",
-        "zicht/standards-php": "*@stable"
+        "zicht/standards-php": "^4"
     },
     "autoload": {
         "psr-0": {
-            "": [
+            "Zicht\\Bundle\\FrameworkExtraBundle": [
                 "src/"
             ]
         }
     },
     "autoload-dev": {
         "psr-0": {
-            "": [
+            "ZichtTest\\Bundle\\FrameworkExtraBundle": [
                 "tests/"
             ]
         }
     },
-    "minimum-stability": "stable",
     "license": "MIT",
     "scripts": {
         "lint": [
-            "./vendor/bin/phpcs -s src/Zicht/ --standard=vendor/zicht-standards/php/Zicht --extensions=php"
+            "phpcs --standard=vendor/zicht/standards-php/phpcs.xml src/ tests/"
+        ],
+        "lint-no-warn": [
+            "phpcs -n --standard=vendor/zicht/standards-php/phpcs.xml src/ tests/"
+        ],
+        "lint-fix": [
+            "phpcbf --standard=vendor/zicht/standards-php/phpcs.xml src/ tests/"
         ],
         "test": [
-            "php7.1 ./vendor/phpunit/phpunit/phpunit -c phpunit.xml.dist"
+            "./vendor/bin/phpunit tests/"
```

Comment:
`./vendor/bin/` mag hier ook weg

Reply:
Hoe weet je dan welke phpunit je gebruikt? Je wilt toch in alle gevallen de phpunit (versie) gebruiken die door deze bundle wordt geinstalleerd? Die bevindt zich in die map

Reply:
Commands worden door Composer automatisch opgezocht in `vendor/bin/`. Zie ook de `lint` commands